### ISSUE TITLE
Holograms with certain materials do not draw in the correct order

### ISSUE THREAD

**Issue by Advers** [thegrb93/StarfallEx#1612]
Holograms do not always draw in the correct order relative to each other.  Specifically what I and some others have discovered is that materials with _both_ the $additive and $translucent flags do not render over any opaque holograms behind them. (Materials that have only one of these flags seem to render correctly.)  This pertains to both server- and client-side holograms.

May be more complex (do other combinations of material parameters cause similar issues?)

**Comment by thegrb93** [comment]
This belongs in garrysmod-issues or something

**Comment by thegrb93** [comment]
Do you have a test chip we can use to see the issue?

**Comment by Advers** [comment]
I will make one right now.

**Comment by Advers** [comment]
It appears that I was mistaken about something before -- I'm not sure how I got the idea, but in actuality, it seems that all additive and all translucent materials (and not just those with the translucent flag) suffer from this issue. I'm not sure where I got the idea before that it was only the combination of the two. I may have mixed up the translucent flag with the alpha flag, or maybe i had a different backdrop (hologram on top of a prop vs hologram on top of a hologram)
![image](https://github.com/thegrb93/StarfallEx/assets/48806866/254666b6-7b95-47ed-a99a-120ee51ef874)
Top row: Additive
Middle row: Translucent
Bottom row: Additive + Translucent
Left column: Holograms, but with RENDERFX DISTORT, which (alongside RENDERFX HOLOGRAM), is the only thing i've discovered which actually fixes the draw order)
Middle column: Holograms
Right column: Props

I should have studied the issue more before, it runs deeper than I thought. Here is the test chip's file: https://pastebin.com/HSKMeb9r
Results are the same regardless of what realm the holograms are created in.
